pkg/apis/openebs.io/v1alpha1/register.go

```diff
@@ -17,10 +17,12 @@ func Resource(resource string) schema.GroupResource {
 }
 
 var (
+	// SchemeBuilder initialization
 	SchemeBuilder runtime.SchemeBuilder
 	// localSchemeBuilder and AddToScheme will stay in k8s.io/kubernetes.
 	localSchemeBuilder = &SchemeBuilder
-	AddToScheme        = localSchemeBuilder.AddToScheme
+	// AddToScheme initialization
+	AddToScheme = localSchemeBuilder.AddToScheme
```

Comment:
@cartinez can you change given comments to something like this 

```suggestion
	// SchemeBuilder is the scheme builder with scheme init functions to run for this API package
	SchemeBuilder = runtime.NewSchemeBuilder
	// AddToScheme is a global function that registers this API group & version to a scheme
	AddToScheme = SchemeBuilder.AddToScheme

```

Reply:
@cartinez you have to revert the previous commit as `SchemeBuilder = runtime.NewSchemeBuilder` added 2 times